src/ui/lib/components/zarf-chip.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
```

Comment:
SPDX + issue to Unicorn UI

Reply:
https://github.com/defenseunicorns/UnicornUI/issues/199